.github/workflows/daily-build.yml

```
@@ -0,0 +1,28 @@
+name: Ballerina datamapper compiler extension daily build
+
+on:
+  schedule:
+    - cron:  '0 */12 * * *'
+
+jobs:
+  ubuntu_build:
+    name: Build on Ubuntu
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
+        env:
+          packageUser: ${{ github.actor }}
+          packagePAT: ${{ secrets.BALLERINA_STDLIB_TOKEN }}
+        run: |
+          export packageUser=packageUser
+          export packagePAT=packagePAT
+          ./gradlew build
```

No need to export environment variables as it is handled by the `env:` block above
```suggestion
        run: ./gradlew build
```